tasks/upload.py

```diff
@@ -552,22 +529,16 @@ def run_impl_within_lock(
             if checkpoints:
                 checkpoints.log(UploadFlow.INITIAL_PROCESSING_COMPLETE)
                 checkpoints.log(UploadFlow.NO_REPORTS_FOUND)
-            log.info(
+            self.log.info(
                 "Not scheduling task because there were no arguments were found on redis",
-                extra=dict(
-                    repoid=commit.repoid,
-                    commit=commit.commitid,
-                    argument_list=argument_list,
-                ),
             )
         return {"was_setup": was_setup, "was_updated": was_updated}
 
-    def fetch_commit_yaml_and_possibly_store(self, commit, repository_service):
+    def fetch_commit_yaml_and_possibly_store(self, commit: Commit, repository_service):
```

Comment:
If you'd like to type `report_service`, that's TorngitBaseAdapter 👍

Reply:
Ah nice, that is good to know! I found out that `PreProcessUpload.fetch_commit_yaml_and_possibly_store` is a duplicated version of this function, so I moved it and its associated tests over to `services.repository`, as it was also not dependent on `self` at all.